tests/entities/test_collection.py

```diff
@@ -0,0 +1,151 @@
+import unittest
+
+from bibx._entities.article import Article
+from bibx._entities.collection import Collection
+
+
+class TestCollection(unittest.TestCase):
```

Comment:
You can skip the `unittest.TestCase` here and run the tests using `pytest`. I think the command is `hatch run pytest`.

Reply:
Ok :+1: